strider/fetcher.py

```diff
@@ -227,12 +230,17 @@ async def generate_from_kp(
 
                 # pin nodes
                 for qnode_id, bindings in result.node_bindings.items():
-                    if qnode_id not in subqgraph["nodes"]:
+                    if qnode_id not in populated_subqgraph["nodes"]:
                         continue
-                    subqgraph["nodes"][qnode_id]["ids"] = (
-                        subqgraph["nodes"][qnode_id].get("ids") or []
-                    ) + [binding.id for binding in bindings]
-                qnode_ids = set(subqgraph["nodes"].keys()) & set(
+                    # add curies from result into the qgraph
+                    # need to call set() to remove any duplicates
```

Comment:
Would this ever cause the curies in the original query graph to be changed? I'm pretty sure that it would stay with the particular subqgraph, but I just want to make sure we wouldn't end up changing the original query graph, since I don't think we want to do that.